src/components/SingleDatePicker.jsx

```diff
@@ -344,7 +620,8 @@ export default class SingleDatePicker extends React.Component {
   }
 
   isHovered(day) {
-    return isSameDay(day, this.state.hoverDate);
+    const { hoverDate } = this.state || {};
```

Comment:
When this function is called from the constructor, `this.state` has not yet been set.